.travis.yml

```diff
@@ -2,5 +2,6 @@ language: node_js
 node_js:
   - "node"
   - "6"
+before_install: if [[ `npm -v` != 5* ]]; then npm i -g npm@5; fi
```

Comment:
this will be weird when npm 6 comes out. can you just always install npm latest instead?